src/style.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500;600;700&display=swap');
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+}
+
+body,
+main,
+section {
+  width: 100%;
+}
+
+ul {
+  list-style: none;
+}
+
+body {
+  background-color: #f2f2f2;
+  font-size: 14px;
+  color: #252525;
+  font-style: normal;
+  font-weight: 400;
+  font-family: 'Open Sans', sans-serif;
+}
+
+main {
+  display: flex;
+  justify-content: center;
+  padding: 5rem 5% 3rem 5%;
+}
+
+#app {
+  background-color: #fff;
+  box-shadow: 0 1px 10px rgba(0, 0, 0, 0.2);
+}
+
+#app #title,
+#app #to-do-list .task {
+  padding: 1rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid #d7d7d7;
+}
```

Comment:
- Hey, great job with your design, however, there is to much space here, kindly update your styling. see the demo video to fix the issue below to improve the user experience of your app.

![image](https://github.com/NitBravoA92/project-to-do-list/assets/59397159/703f24dc-460c-4876-bdc3-74b083828e77)